util.go

```diff
@@ -153,3 +161,32 @@ func containsPeer(list []peer.ID, peer peer.ID) bool {
 	}
 	return false
 }
+
+func BackupState(baseDir string, state state.State) error {
```

Comment:
I think we can pass *Config here. This function can probably be improved with a "backup retain number" which will be another Config key and who knows what else. So more versatile to just receive the Config.